### PR TITLE
Improve Update zone NS records API docs

### PR DESCRIPTION
**Pull request by lokst**
Add information about vanity name servers

### CODE REVIEW

File: content/v2/zones/ns-records.markdown [hunk 1]
```diff
@@ -18,7 +18,7 @@ excerpt: This page documents the DNSimple domain name servers API v2.
 
     PUT /:account/zones/:zone/ns_records
 
-Update the NS records for a zone in the account.
+Update the NS records for a zone in a hosted domain in the account. If vanity name servers are enabled for the domain, plesae ensure that they are included in the name server name parameters or referenced name server sets, for the vanity name server configuration to stay effective.
```
Comment (kojdm) [inline content/v2/zones/ns-records.markdown:21]:
```suggestion
Update the NS records for a zone in a hosted domain in the account. If vanity name servers are enabled for the domain, please ensure that they are included in the name server name parameters or referenced name server sets, for the vanity name server configuration to stay effective.
```

Comment (kojdm) [reply content/v2/zones/ns-records.markdown:21]: WDYT wrapping this in `<info>` or `<warning>` tags for emphasis?

```
Update the NS records for a zone in a hosted domain in the account

<info>
If vanity name servers are enabled for the domain, please ensure...
</info>
```

**Review by kojdm — APPROVED** [review]
LGTM!

File: content/v2/zones/ns-records.markdown (hunk 1 above)
Comment (lokst) [reply content/v2/zones/ns-records.markdown]: Great idea! Wrapped the info in `<info>` tags 👍

**Review by dallasread — APPROVED** [review]

**Pull request merged**